src/components/landing/listing.js

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+import Image from "next/image";
+import styles from "./listing.module.css";
```

Comment:
This is the CSS approach that I am learning in my React course 😅  I like the approach.